### DISCUSSION TITLE
frame of reference

### DISCUSSION THREAD

**Discussion by joaofrancafisica** [sncosmo/extinction#21] [Q&A] [answered]
Hello!

I have a simple doubt: Do I have to deredshift a spectra from observer frame in order to use these extinction models?

Thank you!

**Comment by kbarbary** [comment, marked as answer]
Probably yes (or you could shift the wavelengths of the extinction values).

These extinction models are in the rest-frame of the dust, so if you're modeling dust around the object of interest, you should either deredshift the object spectrum or redshift the extinction values.

The exception is that if you're modeling milky way dust, then the dust is in the observer reference frame, so no redshifting / deredshifting is required.
